bin/v-backup-user

```diff
@@ -31,6 +31,106 @@ source $HESTIA/func/backup.sh
 # shellcheck source=/usr/local/hestia/conf/hestia.conf
 source $HESTIA/conf/hestia.conf
 
+# Return Disk Usage
+get_user_disk_usage() {
+    u_usage=0
+
+    web_exclusions=$(grep 'WEB=' $USER_DATA/backup-excludes.conf |\
```

Comment:
By default backup-excludes.conf doesn't exists we should check if it exits  and if not we should give calculate the current disk usage.

Reply:
further down the file the way the file is read is by sourcing the backupfiles (if exists). Reason I didn't go that route is because during my tests with the method as a standalone, $USER wouldnt be "USER", but my current user. I was unsure if this reliably works inside it script. If we can assure the script works, then we can just source the file, and replace the variables.